source/cloudserver/webapi/XPowerApi/XPowerApi/Models/HubModels/HubCreate.cs

```diff
@@ -0,0 +1,14 @@
+namespace XPowerApi.Models.HubModels;
+
+public class HubCreate
+{
+    public string Name { get; set; }
+
+    public string Mac { get; set; }
+    
+    public string Home { get; set; }
```

Comment:
when creating a hub, do we know the name of it and in what home it belongs?

Reply:
i would assume yes?